ci-operator/step-registry/osd-ccs/cluster/provision/gcp/osd-ccs-cluster-provision-gcp-commands.sh

```diff
@@ -68,6 +68,40 @@ function add_iam_policy_binding()
   done
 }
 
+CLUSTER_MACHINES_CREATED=false
+function wait_for_bootstrap() {
+  if ${CLUSTER_MACHINES_CREATED}; then
+    # There are already compute instances of the cluster on GCP.
+    return 0
+  fi
+
+  if ! which gcloud; then
+    GCLOUD_TAR="google-cloud-sdk-468.0.0-linux-x86_64.tar.gz"
+    GCLOUD_URL="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/$GCLOUD_TAR"
+    logger "INFO" "gcloud not installed, installing from $GCLOUD_URL"
+    pushd ${HOME}
+    curl -O "$GCLOUD_URL"
+    tar -xzf "$GCLOUD_TAR"
+    export PATH=${HOME}/google-cloud-sdk/bin:${PATH}
+    popd
+  fi
+
+  # login to the service project
+  service_project_id="$(jq -r -c .project_id "${GCP_CREDENTIALS_FILE}")"
+  gcloud auth activate-service-account --key-file="${GCP_CREDENTIALS_FILE}"
+  gcloud config set project "${service_project_id}"
```

Comment:
Could we write lines 78-92 into a gcloud login function?

Reply:
It's done in https://github.com/openshift/release/pull/50326, thanks!